### ISSUE TITLE
Add kafka support 

### ISSUE THREAD

**Issue by chjohnst** [pyr/cyanite#42]
I think this would bring cyanite closer to the features of carbon with the exception of python pickle format.  We are looking at use cases of sending data to brokers to fan out messaging to several cyanite instances.

**Comment by SEJeff** [comment]
Indeed, kafka would be perfect. A stack of kafka, cassandra, storm, etc, would be super nice.
